tsconfig.json

```diff
@@ -10,7 +10,7 @@
         "target": "es5",
         "lib": [
             "es6",
-            "dom"
+            "webworker"
```

Comment:
do you know why we need webworker on backend side ?

Reply:
This comes from generated sample plug-in